Rubberduck.CodeAnalysis/Inspections/Concrete/UnderscoreInPublicClassModuleMemberInspection.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Inspections.Abstract;
+using Rubberduck.Inspections.Results;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Resources.Inspections;
+using Rubberduck.Parsing.VBA;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    public sealed class UnderscoreInPublicClassModuleMemberInspection : InspectionBase
+    {
+        public UnderscoreInPublicClassModuleMemberInspection(RubberduckParserState state)
+            : base(state) { }
+
+        protected override IEnumerable<IInspectionResult> DoGetInspectionResults()
+        {
+            var names = State.DeclarationFinder.UserDeclarations(Parsing.Symbols.DeclarationType.Member)
+                .Where(w => w.ParentDeclaration.DeclarationType == Parsing.Symbols.DeclarationType.ClassModule)
+                .Where(w => !State.DeclarationFinder.FindAllInterfaceImplementingMembers().Contains(w))
```

Comment:
I just realized this is also going to false positive for event handlers. Although to some extent that might be inevitable - see #389, #3626, and #3924.

Reply:
Ugh.

Reply:
Consider finally implementing typelib api that will tell you what's what and thus know what's events and what's not. There are several issues that would be closed by this enhancement. [Reference](https://github.com/rubberduck-vba/Rubberduck/projects/15#card-7002230)

Reply:
Sounds good to me. I might get to it today.